buildSrc/src/main/groovy/starter.java.deps-build-conventions.gradle

```diff
@@ -11,9 +11,6 @@ plugins {
 dependencies {
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
 
-    // Added
-    implementation 'javax.validation:validation-api'
```

Comment:
We don't need to import this for all java projects, plus Spring Boot 3 uses `jakarta` instead.